flaskr/tweets.py

```diff
@@ -1,15 +1,19 @@
+import itertools
 import os
+import string
 import openai
 from flask import (
     Blueprint, current_app, g, jsonify, render_template, session, request, redirect, url_for
 )
+from spellchecker import SpellChecker
```

Comment:
I have trouble importing this lib because of its dependence to "indexer" 
We may need to adjust the requirement

Reply:
fixed